skipped_tests.toml

```diff
@@ -76,11 +80,17 @@ unit = [
 	"tests/unit/states/test_boto_apigateway.py::BotoApiGatewayTestCase::test_absent_when_stage_is_valid_and_only_one_stage_is_associated_to_deployment",
 	"tests/unit/states/test_boto_apigateway.py::BotoApiGatewayTestCase::test_absent_when_stage_is_valid_and_two_stages_are_associated_to_deployment",
 	"tests/unit/states/test_boto_apigateway.py::BotoApiGatewayTestCase::test_present_when_stage_associating_to_new_deployment_errored_on_api_creation",
+	"tests/unit/states/test_boto_apigateway.py::BotoApiGatewayTestCase::test_present_when_stage_associating_to_new_deployment_errored_on_lambda_function_lookup",  # missing boto libraries
```

Comment:
This might be fixed by a doing the same I did in https://github.com/openSUSE/salt/pull/616/commits/7961890c8a6a4adc59240347ddc96b9c7a973c7e

Reply:
Right, this is not needed anymore :+1:

Reply:
Yeah, that is true. We can actually get rid of all listed ones for `tests/unit/states/test_boto_apigateway.py`. I'll update the PR in a moment.